azurerm/cluster/configuration.tf

```diff
@@ -17,6 +17,8 @@ locals {
 
   dns_prefix = lookup(local.cfg, "dns_prefix", "api")
 
+  sku_tier = lookup(local.cfg, "sku_tier", "Free")
```

Comment:
You have to also set the `sku_tier` variable to `local.sku_tier` in `cluster/main.tf`. Otherwise it will only ever use the default value from the variable definition above.

Reply:
Thanks, done :)